usaspending_api/recipient/management/sql/restock_summary_award_recipient.sql

```diff
@@ -0,0 +1,83 @@
+--------------------------------------------------------------------------------
```

Comment:
If we keep this as a restock, we could same additional time by building them from materialized views. This script could then be run in parallel with the restock_recipient_profile script during the nightly pipeline after the matviews are recreated/refreshed.